core/src/main/java/org/frankframework/align/Json2Xml.java

```diff
@@ -48,6 +48,7 @@
 import org.apache.commons.lang3.StringUtils;
 import org.apache.commons.lang3.builder.ToStringBuilder;
 import org.apache.commons.lang3.builder.ToStringStyle;
+import org.apache.commons.lang3.tuple.Pair;
```

Comment:
I personally like to create a dataclass instead of a pair for readability

Reply:
That is actually a good idea. Done.